app/views/pages/policies.md

```diff
@@ -0,0 +1,6 @@
+# RubyGems.org Policies
+
+* [Ruby Central Privacy Policy](https://rubycentral.org/privacy-notice)
+* [Privacy Policy](/policies/privacy-notice)
```

Comment:
These two stood out. Maybe it's important to explain why one would go read the RC privacy policy? Otherwise it's not clear which to click (and since this is rubygems, should it come first?)

Reply:
This page needs a narrative.  I'll make sure to provide that before it's done.  I appreciate the callout on the confusion between the two policies.  The Ruby Central one is the primary privacy notice, and the RubyGems.org one is specific to just RubyGems.org.  I think I can remove the RC link as it's linked with the RG one.